spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -252,7 +252,7 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
    * @param input input data
    * @return output data
    */
-  final def forward(input: A): B = {
+  final def forward(input: A = null.asInstanceOf[A]): B = {
```

Comment:
Remove it to avoid confusion

Reply:
This is the main change of this PR.
Constant layer requires no input, which mean the forward method here needs to take no input, the input needs to be null.

Reply:
This is just default value, it is not mandatory that the input has to be null. Makes no difference here

Reply:
The difference is between forward(null) and forward().
First case, forward still takes input, and the input is null.
The second case, forward takes no input.